README.md

```diff
@@ -7,31 +7,95 @@
-|                                                               **[How to setup](./packages/cli/README.md#getting-started)** a basic project                                                               |                                                             Sort, filter **[your goals](./packages/cli/README.md#portal-integration)**                                                             |                                                   Updates **[on every PR](./packages/cli/README.md#-ci-automation)**                                                   |
-| <a href="#getting-started"><img alt="Getting started cover image" title="Getting started with code-pushup" src="./packages/cli/docs/images/getting-started-cover.png" max-height="200" width="auto"></a> | <a href="#portal-integration"><img alt="Portal integration cover image" title="Inetegrate code-pushup portal" src="./packages/cli/docs/images/portal-cover.png" max-height="200" width="auto"></a> | <a href="#-ci-automation"><img alt="CI Automation cover" title="CI automation guide" src="./packages/cli/docs/images/cli-cover.png" max-height="200" width="auto"></a> |
+<h3>Zero config setup</h3>
+
+- [📦 @code-pushup/cli](./packages/cli#readme)
+  - **CLI** for **collecting** audit results and **uploading** report to portal.
+  - [Get started in no time!](./packages/cli/README.md#getting-started)
 
 ---
 
-This monorepo contains code for open-source Code PushUp NPM packages:
+<h3>Officially supported plugins </h3>
+
+- 🧩 [@code-pushup/eslint-plugin](./packages/plugin-eslint#readme)
+  - Static analysis using **ESLint** rules
+- 🧩 [@code-pushup/coverage-plugin](./packages/plugin-coverage#readme)
+  - **Test Coverage** analysis
+- 🧩 [@code-pushup/js-packages-plugin](./packages/plugin-js-packages#readme)
+  - Package audit on **Security** and **Outdates**
+- 🧩 [@code-pushup/lighthouse-plugin](./packages/plugin-lighthouse#readme)
+  - web performance and best practices from **Lighthouse**
+
+---
+
+<h3>Custom Plugins</h3>
+
+- 🛠️ [Integrate your own metric with custom plugins](./packages/cli/README.md#getting-started)
+  - Example plugins repo
+  - Docs
+- 📦 [@code-pushup/models](./packages/models#readme)
+  - **schemas and types** for data models (useful for custom plugins or other integrations
+- 📦 [@code-pushup/utils](./packages/utils#readme)
+  - various **utilities** (useful for custom plugins or other integrations)
+  <!-- [Get enterprise support](https://code-pushup.dev/enterprise-support) -->
+
+<h2>How It Works!</h2>
+
+<ol>
+<li>
+<b>Configure</b><br/>
+Pick form a set of supported packages or include your own ideas.
+</li>
+<li>
+<b>Integrate</b><br/>
+Use our integration guide and packages to setup CI integration in minutes.
+</li>
+<li>
+<b>Observe</b><br/>
+Guard regressions and track improvements with every code change.
+</li>
+<li>
+<b>Wing it!</b><br/>
+Watch improve, share reports
+</li>
+</ol>
```

Comment:
Why is everything using HTML tags instead of normal Markdown formatting?

Reply:
formatted